src/api/python/CMakeLists.txt

```diff
@@ -17,6 +17,7 @@ if(NOT ONLY_PYTHON_EXT_SRC)
   # Python modules for building and installing
   check_python_module("setuptools")
   find_package(Cython 3.0.0 REQUIRED)
+  find_package(Repairwheel 0.2.9 REQUIRED)
```

Comment:
Can you add a description that explains why repairwheel is required?

Reply:
Done